rfcs/xxxx-standard-gh-labels.md

```diff
@@ -0,0 +1,118 @@
+---
+created-on: 2021-03-23   (date the RFC was created)
+freshness: YYYY-MM-DD    (date the RFC was last reviewed for freshness)
+owner: @CalvinR (owner responsible for freshness review)
+status: draft
+tags:
+  - github
+  - project_management
+---
+
+[La version française suit.](#résumé)
+
+# CDS Github Label Standard
+
+## Summary
+Create an organization wide standard for Github labels to ensure a consistent way of identifying issues across the org as well as allow customization of labels to happen at the business unit level.
+
+## Motivation
+
+Currently every github repo has to modify existing Github labels upon creation of a new repo or use the default labels. Unfortunately GH does not provide bilingual labels out of the box and so we are non-compliant with OLA (Official Languages Act) requirements. If we customize lables for a business unit (BU) we will then have to manually copy labels into every repo that is owned by that BU.
+
+## Proposal
+A standard baseline of labels to be applied across the cds-snc github organization. This standard will be enforced through a github action on each repo, and will allow for a consistent way to identify issues across the organization and will allow for products, business units or specific products to extend this baseline with labels that fit their needs. This will also allow for CDS to ensure that we are respecting our requirements to communicate with people in their official language of choice.
+
+See https://github.com/cds-snc/covid-alert-server-labels for an example of what is being proposed to implement this solution. 
+
+The source of truth for these labels will reside in github and what is put forth in this document is just to start the discussion. The understanding is that if this Standard is accepted that these labels will be copied into github and maintenance and discussion will continue in those repositories. 
```

Comment:
What do you mean by 'copied into github'? We'll create a fork of the labels repo above and they'll be housed there? Or, a discussion?

Reply:
I mean a github repo will be created to manage this action, I'll clarify text